### PR TITLE
Update Gas Station + Relayer Server Example

### PR DESCRIPTION
**Pull request by bucanero**

### CODE REVIEW

File: docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md [hunk 1]
```diff
@@ -11,24 +11,34 @@ In this article you'll learn how to run end-to-end tests on the entire Multichai
...
 :::
 
+## Integration test
+
+In separate terminals, you need to run the following tools:
+
+1. [Multichain Relayer server](#multichain-relayer-server), with a valid `config.toml` configuration file
+2. [Gas Station indexer](#gas-station-event-indexer), with correct values in `config.toml`
+3. [Python integration test script](#run-integration-test)
+
```
Comment (anthony-near) [inline docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md:41]: I think we should call out that this is the best way to test that all multichain gas relayer systems are working well together and the Manual Testing section below is a good way to debug if any individual part of the system isn't working

File: docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md [hunk 2]
```diff
@@ -11,24 +11,34 @@ In this article you'll learn how to run end-to-end tests on the entire Multichai
...
+ - [NEAR CLI RS](https://github.com/near/near-cli-rs): Make sure to configure it with the correct network and account.
 
 :::
 
 ## Setup
 
-Before you start testing, set up your local environment and install the Relayer server, the Event indexer and NEAR CLI.
+Before you start testing, set up your local environment and install the [Relayer server](multichain-server.md), the [Event indexer](https://github.com/near/gas-station-event-indexer) and [NEAR CLI](https://github.com/near/near-cli-rs).
 
 :::info Alpha stage
```
Comment (anthony-near) [inline docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md:28]: Code audit it complete. Think we can remove this section

**Review by anthony-near — CHANGES_REQUESTED** [review]
Looks good. Just added some thoughts that weren't explicitly in the google doc

File: docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md (hunk 2 above)
Comment (bucanero) [reply docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md:28]: Removed ✅

File: docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md (hunk 1 above)
Comment (bucanero) [reply docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md:42]: Fixed

**Review by anthony-near — APPROVED** [review]

**Pull request merged**